README.md

```diff
@@ -37,7 +37,7 @@ fn main() {
    journal::send(["CODE_FILE=HI", "CODE_LINE=1213", "CODE_FUNCTION=LIES"]);
    journal::JournalLog::init().unwrap();
    warn!("HI");
-   sd_journal_log!(4, "HI {:?}", 2);
+   sd_journal_log!(log::Level::Warn, "HI {:?}", 2);
```

Comment:
I'd really prefer to be able to pass the actual integer here still, so I suppose that means I'd prefer at least some of the logging functions be preserved.